biz.aQute.tester.junit-platform/src/aQute/tester/junit/platform/Activator.java

```diff
@@ -209,6 +210,16 @@ public void executionSkipped(TestIdentifier testIdentifier, String reason) {
 				message("", "TEST %s <<< SKIPPED", testName(testIdentifier));
 			}
 		});
+		ServiceTracker<TestExecutionListener, TestExecutionListener> track = new ServiceTracker<>(context,
+			TestExecutionListener.class, null);
+		track.open();
+		Optional.ofNullable(track.getServices())
+			.ifPresent(services -> Stream.of(services)
+				.map(TestExecutionListener.class::cast)
+				.forEach(listener -> {
+					listenerList.add(listener);
+				}));
+		track.close();
```

Comment:
You revert back to the bad behavior of closing the tracker before using the tracked services.

Reply:
Oops, something went wrong with my git pull before i made the change. Will have to fox it tomorrow now... sorry!

Reply:
Fixed.